continuous-integration/windows/setup.py

```diff
@@ -42,9 +42,51 @@ def _install_zivid_sdk():
         _run_process((str(zivid_installer), "/S"))
 
 
+def _install_intel_opencl_runtime():
+    import requests  # pylint: disable=import-outside-toplevel
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        intel_opencl_runtime_url = "https://www.dropbox.com/s/09bk2nx31hzrupf/opencl_runtime_18.1_x64_setup-20200625-090300.msi?raw=1"
+        print("Downloading {}".format(intel_opencl_runtime_url), flush=True)
+        opencl_runtime = Path(temp_dir) / "opencl_runtime.msi"
+        response = requests.get(intel_opencl_runtime_url)
+        opencl_runtime.write_bytes(response.content)
+        print("Installing {}".format(opencl_runtime), flush=True)
+        _run_process(("msiexec", "/i", str(opencl_runtime), "/passive"))
+
+
+def _write_zivid_cpu_configuration_file():
+    content = """__version__: 4
+Configuration:
+  ComputeDevice:
+    SelectionFilter: ""
+    Type: CPU
+    Vendor: Any
+  DriverConfig:
+    ToshibaTeli:
+      VersionCheck: yes
+  Logging:
+    Asynchronous: yes
+    CrashHandler: yes
+    Directory: ""
+    Enabled: yes
+    FlushOnExit: yes
+    HistorySize: 30
+    MaxFileSizeMB: 50
+    MaxTotalSizeMB: 1000
+    RotationCount: 2
+    WriteInterval: 0
+"""
```

Comment:
Can't you use "ZividAPIConfig.yml" like we do in the Linux setup?

Reply:
Good point, did not think about this